### PR TITLE
Update CHANGELOG for v15

### PR DESCRIPTION
**Pull request by dbrgn**

### CODE REVIEW

**Review by rnestler — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,29 @@ Changes should start with one of the following tags:
 - `[changed]` for changes to the existing API
 - `[removed]` for keys that have been removed from the schema
 
+## v15 (unreleased)
+
+Root level:
+
+- [added] The `linked_spaces` field was added ([#75])
```
Comment (rnestler) [inline CHANGELOG.md:15]: Would you add links to all PRs which are about `linked_spaces`? https://github.com/SpaceApi/schema/pull/92 for example?

**Review by rnestler — DISMISSED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (dbrgn) [reply CHANGELOG.md:15]: Ah, good catch. If there's an issue and a PR, I'd only reference one of them, but if there are multiple PRs, then all of them should be listed of course.

**Review by s3lph — APPROVED** [review]

**Comment by dbrgn** [comment]
UGH, reviews are auto-dismissed after force pushes, even when just rebasing.

I unchecked "Dismiss stale pull request approvals when new commits are pushed" in the settings and reduced the number of required approvals to 1 (we'll still use common sense before clicking merge).

@s3lph would you care to re-approve after my rebase?

**Comment by dbrgn** [comment]
Never mind, I temporarily disabled the required approvals, since we already got 2 for this PR.

**Pull request merged**